src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -22,6 +22,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
     internal partial class NamedParameterCompletionProvider : CommonCompletionProvider, IEqualityComparer<IParameterSymbol>
     {
         private const string ColonString = ":";
+        private static readonly CompletionItemRules CompletionItemRules = CompletionItemRules.Default
```

Comment:
should be s_rules.  s_ is needed,but the suffix is up to you.  This is also because it makes it much omre confusing to read CompletionItemRules below.  It's not at all clear that these are special custom rules for this provider.